nnstreamer_example/custom_example_python/nnstreamer_customfilter_python_interface.c

```diff
@@ -0,0 +1,470 @@
+/**
+ * NNStreamer Custom Filter Interface for Python
+ * 
+ * Copyright (C) 2019 Dongju Chae <dongju.chae@samsung.com>
+ *
+ * LICENSE: LGPL-2.1
+ *
+ * @file  nnstreamer_customfilter_python_interface.cc
+ * @date  26 March 2019
+ * @brief NNStreamer Custom Filter Interface for Python
+ * @author  Dongju Chae <dongju.chae@samsung.com>
+ * @bug   No known bugs
+ * 
+ * This example provides an interface to access custom filters written in Python
+ *
+ */
+
+#include <stdlib.h>
+#include <stdio.h>
```

Comment:
Do we need stdio.h?

Reply:
Thanks for the comment! It was for the debugging purpose. 
I will port all codes as a sub-plugin and remove it as well.

Reply:
You may use glib instead of std*.